src/igvfd/schemas/changelogs/prediction.md

```diff
@@ -1,5 +1,9 @@
 ## Changelog for *`prediction.json`*
 
+### Minor changes since schema version 3
+
+* Make required properties `lab` and `award` not submittable. No further submission of predictions are accepted, please submit prediction sets instead.
```

Comment:
* Restrict `lab` and `award` to admins only.

Reply:
Added.